packages/Input/Input.jsx

```diff
@@ -60,7 +64,7 @@ Input.propTypes = {
   /**
    * An error message. Either an error or a helper should be used, not both.
    */
-  error: PropTypes.string,
+  error: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
Let's change `PropTypes.node` to `componentWithName('Link')` to restrict this to text and the core-link component.

Reply:
But we need to allow a combination of strings **and** Link: https://github.com/telus/tds-core/issues/894#issuecomment-455325223

Seems like [componentWithName](https://github.com/telus/tds-core/blob/367747116d1afcd7b3be304db84c7846093c267e/packages/prop-types/PropTypes.md#componentwithnamestring) would force the whole content to be a `Link`

This could be updated to use `or` though:

> `or([PropTypes.string, PropTypes.node])`

Reply:
Can we use `componentWithName('Link')` here and in FormField?

Reply:
Ok, will update to:
> PropTypes.oneOfType([PropTypes.string, componentWithName('Link')])

Reply:
Updating to `error: or([PropTypes.string, componentWithName('Link')])` will cause the following warning:
<img width="489" alt="error" src="https://user-images.githubusercontent.com/4450690/53264432-90e7e400-36a9-11e9-8f94-f9f92179148d.png">

When used with the following examples in https://github.com/telus/tds-core/issues/894, e.g.:
```<Input error={<React.Fragment>Hello <Link href="#">What is this</Link></React.Fragment>}  />```

Are we not allowing this format of messages?

Reply:
Please rewrite this as `error: PropTypes.node,` as we'll accept all children for now, combined with additional usage criteria to guide designers and developers on their use of the `error` prop.